Source/Android/app/src/main/java/org/dolphinemu/dolphinemu/activities/EmulationActivity.java

```diff
@@ -906,7 +906,7 @@ private void chooseController()
                       getResources().getStringArray(R.array.controllersValues)[indexSelected]);
               NativeLibrary.SetConfig(SettingsFile.FILE_NAME_WIIMOTE + ".ini",
                       SettingsFile.KEY_WIIMOTE_PLAYER_1, SettingsFile.KEY_WIIMOTE_ORIENTATION,
-                      indexSelected == InputOverlay.OVERLAY_WIIMOTE_SIDEWAYS ? "True" : "False");
+                      indexSelected == InputOverlay.OVERLAY_WIIMOTE_HORIZONTAL ? "1" : "0");
```

Comment:
Why is this using 1 and 0 instead of True and False now? I'm assuming there's some reason, but it isn't obvious to me.

Reply:
The "Orientation" setting looks nicer as a `SingleChoiceSetting` (int values), rather than a `CheckBoxSetting` (boolean value) with a description to explain default orientation. It seems silly to convert `int` -> `boolean` -> `int`, so I think it's best jut to change the values of this setting to `int`.

Reply:
But it's still a bool in the native code. If the native code re-saves the INI, it can replace the 1 and 0 with True and False, and the updated Android code can't parse that correctly.

Reply:
Actually, now that you've renamed the setting, I suppose this doesn't correspond to any native setting at all anymore. So if you set the overlay controls to Sideways Wii Remote, the d-pad and motion controls will be oriented incorrectly.

Reply:
Alright, do you think it would be best to keep `KEY_WIIMOTE_ORIENTATION`'s bool value and implement a `int` -> `boolean` -> `int` conversion so `SingleChoiceSetting` can be used over `CheckBoxSetting` for better UX design?

Reply:
I would be fine with either that or making it a checkbox.

Reply:
Reverted the `KEY_WIIMOTE_ORIENTATION` rename and made it 'boolean' again. I also moved "Catch getSetting ClassCastExceptions in ItemViews" to its own comment in case INI settings come across invalid values.

Still working on the rest of the review comments and the bug I found.